### DISCUSSION TITLE
Gestione inserimento episodi

### DISCUSSION THREAD

**Discussion by bionicco** [pdonadeo/e-uscito-joypad#11] [General]
Questo thread serve per annunciare quali episodi si stanno per inserire nel backend, così evitiamo di fare duplicati e possiamo capire su quali lavorare, direi di prenotare a "blocchi" in modo da non dover ocntrolalre continuamente cosa viene scritto

**Comment by bionicco** [comment]
per ora faccio 30-35

**Reply by agofa** [reply]
Dovrebbero essere tutti ora, senza doppioni mi sembra
Come detto, io ho messo la URL de Il post con il file audio e come prova, nell'episodio 8, immagine presa da instagram e ridimensionata, ma non so come sia venuta

**Comment by agofa** [comment]
Con tutto il ritardo del mondo, ho scoperto da poco che ci sono due URL al podcast: una con la sola registrazione e una con anche del testo
Quale deve essere usata? Io ho usato solo la prima, perché ho scoperto da poco l'esistenza della seconda.

**Reply by pdonadeo** [reply]
> Con tutto il ritardo del mondo, ho scoperto da poco che ci sono due URL al podcast: una con la sola registrazione e una con anche del testo Quale deve essere usata? Io ho usato solo la prima, perché ho scoperto da poco l'esistenza della seconda.

Intendi la URL del video?

![image](https://user-images.githubusercontent.com/242049/167580322-df6df5f1-fe58-4cd0-b12d-a82ca1eda7d6.png)


L'ho aggiunta solo ieri sera, per forza che non l'avevi vista 😂 

Vedi: https://github.com/pdonadeo/e-uscito-joypad/discussions/10#discussioncomment-2717638

Sembra che alcuni episodi del podcast siano disponibili in formato video, vedi [questi messaggi](https://github.com/pdonadeo/e-uscito-joypad/discussions/10#discussioncomment-2710274) e relativa risposta di Zampa.

**Reply by bionicco** [reply]
Mi sa che il dubbio è lo stesso che avevo espresso io [qui](https://github.com/pdonadeo/e-uscito-joypad/discussions/10#discussioncomment-2709856)

Su ilpost c'è una pagina con tutti gli episodi di joypad, in cui selezionandone uno singolo si ottine un url, fatto così (es: ep48):
https://www.ilpost.it/episodes/ep-48-quello-con-le-console-con-il-deserto-intorno/
però esiste anche la pagina specifica dell'episodio, con la descrizione testuale (sempre ep48):
https://www.ilpost.it/2022/04/30/la-quarantottesima-puntata-del-podcast-di-joypad/

Io ho preferito inserire quelli del secondo tipo, perchè mi sembravano più ricchi di informazioni (nonchè leggermente più difficili da reperire rispetto al primo tipo, quindi più utile averli a disposizione).
Decidiamo quali usare in modo da avere uniformità nei dati

**Reply by pdonadeo** [reply]
Ok, non avevo capito niente allora: vi riferite ai post che annunciano la puntata.

Sono cose diverse, forse ci vuole un altro campo URL.

**Comment by pdonadeo** [comment]
Ho aggiunto "tutte le URL del mondo" 😂 

![image](https://user-images.githubusercontent.com/242049/167588936-42cdb03d-49ac-48cf-9189-69d7baaca86c.png)

La presenza o meno di questi campi valorizzati è presente nella _list view_:

![image](https://user-images.githubusercontent.com/242049/167589135-9b356267-8bb7-493d-8bed-36278ff4afde.png)

**Reply by bionicco** [reply]
Perfetto, aggiornerò (intanto sto inserendo i video, pensavo fossero pochi invece sono una trentina! quanto mi mancano!)

**Reply by agofa** [reply]
> Ho aggiunto "tutte le URL del mondo" 😂

Ah bene ;-)

E che nessuno si lamenti più della scarsità di URL

Stasera potrò fare cose, scrivetemi qui cosa c'è da fare, per evitare doppioni

**Comment by agofa** [comment]
Sto aggiungendo le URL dell'annuncio episodio, ma non sono (ancora) riuscito a trovare questa
https://db.euscitojoypad.it/backoffice/episodio/12/change/?_changelist_filters=p%3D2
(episodio extra Quello con i migliori giocattolai del mondo)

**Reply by agofa** [reply]
> Sto aggiungendo le URL dell'annuncio episodio, ma non sono (ancora) riuscito a trovare questa

Mi rispondo da solo. Dovrei aver finito, non sono riuscito a trovare l'episodio 41 e l'episodio extra del 14/12/2020

Ho chiesto a Zampa

**Comment by bionicco** [comment]
Ok dovrebbero esserci tutti gli episodi (compresi i video)
Mancano un po' di immagini per i video ma ho visto che qualcuno si è divertito per ora a inserirli (grazie) per cui imamgino inserirà anche gli altri...

**Reply by agofa** [reply]
Ho iniziato io, "lento pede" metterò anche le altre

**Reply by agofa** [reply]
Ordunque, mi sono accorto che dal video 9 in poi l'immagine nella pagina del post è il primo fotogramma del video con i nostri 3 baldi eroi. Vabbé, baldi, si fa per dire. E pure su "eroi" ci sarebbe da ridire...
Che si fa? Si usa quell'immagine?

**Reply by pdonadeo** [reply]
> Ordunque, mi sono accorto che dal video 9 in poi l'immagine nella pagina del post è il primo fotogramma del video con i nostri 3 baldi eroi. Vabbé, baldi, si fa per dire. E pure su "eroi" ci sarebbe da ridire... Che si fa? Si usa quell'immagine?

Direi che decide @zampale

**Reply by zampale** [reply]
Posso fare io delle thumb per tutti i video!

**Reply by pdonadeo** [reply]
> Posso fare io delle thumb per tutti i video!

GRAZIE!

**Comment by agofa** [comment]
Ho finito episodio 2, inserimento collegamento a giochi

**Reply by pdonadeo** [reply]
Madonna sei un eroe, l'episodio dei TGA era difficile, ne hanno citati un sacco.

**Reply by agofa** [reply]
Però non ho riportato l'elenco delle candidature

**Reply by pdonadeo** [reply]
Secondo me va benissimo così.

**Comment by bionicco** [comment]
Mi occuperò dei video (sono ababstanza veloci ed in genere monotitolo)

**Reply by bionicco** [reply]
Video fatti (quasi tutti recensione di un unico titolo)

**Reply by agofa** [reply]
Grande!

**Comment by bionicco** [comment]
Aggiunti 4-5-6 con informazioni prese dalla scaletta (quindi incomplete/imprecise)
Ho aggiunto nelle note che le info sono parziali

**Comment by agofa** [comment]
Finito episodio 3, me le vado proprio a cercare: 15, 16, 17 citazioni e oltre....

**Reply by bionicco** [reply]
😮 
(ho aggiunto il gioco sky che non trovavi)
Fifa ultimate dovrebbe essere una modalità di gioco dentro i vari fifa...quindi non un gioco a se...forse si può collegare a una fifa preciso (vista la data dell'episodio direi il 17)

**Reply by bionicco** [reply]
Comunque che bello! Sto giocando the witness e volevo sentire il consiglio del Fossetti che ricordavo avesse dato, ma non sapevo come ritrovarlo...l'archivio funziona già! 😃

**Reply by agofa** [reply]
> (ho aggiunto il gioco sky che non trovavi)

Grazie!
Anche della spiegazione FIFA, non mi interessano i giochi sportivi tranne il baseball (e vabbé ognuno ha i suoi problemi, chissà perché proprio a me sono capitati i miei?)

**Reply by agofa** [reply]
Ho revisionato episodio 4.
Sto usando le note libere per indicare lo stato di avanzamento di un episodio
Direi quindi che 1, 2, 3 e 4 sono completi. Ci vorrà un po' per finire..

**Comment by n1k9** [comment]
Sto facendo l'episodio 7

**Reply by n1k9** [reply]
ho finito ma non trovo in rawg "http://guessthecorrelation.com/" perché è una cosa un po' strana, come si fa in questi casi? comunque ho aggiunto una nota

**Reply by pdonadeo** [reply]
> ho finito ma non trovo in rawg "http://guessthecorrelation.com/" perché è una cosa un po' strana, come si fa in questi casi? comunque ho aggiunto una nota

Boh! Lo salterei e lascerei la nota. Mi sembra un gioco molto molto bizzarro.

**Comment by agofa** [comment]
Io ho preso il 5

**Reply by agofa** [reply]
Finito, facile facile

**Comment by n1k9** [comment]
inizio l'8

**Reply by n1k9** [reply]
Finito

**Reply by pdonadeo** [reply]
@n1k9 finito... cosa?

**Reply by n1k9** [reply]
@pdonadeo essendo in risposta al mio precendete "inizio l'8" intendevo finito l'episodio 8

**Comment by bionicco** [comment]
Ho inserito l'ultimo episodio (49 senza i riferimenti ai contenuti, solo link e titolo), da oggi ufficialmente uscito,
Mi aspettavo che si aggiornasse anche la home, ma probabilmente pesca i dati da un'altra parte...

**Reply by pdonadeo** [reply]
> Mi aspettavo che si aggiornasse anche la home, ma probabilmente pesca i dati da un'altra parte...

Ma quale home non è aggiornata? La home di https://www.euscitojoypad.it/ si è aggiornata eccome.

**Reply by bionicco** [reply]
Benissimo, forse era solo un problema di cache...

**Reply by agofa** [reply]
Io l'ho visto aggiornato

**Comment by n1k9** [comment]
Ciao, nell'episodio 14 si parla di nn developer in modo diffuso  (https://rawg.io/developers/naughty-dog) lo si inserische trai o giochi secondo voi?

**Reply by agofa** [reply]
Io direi di no tra i giochi
Non so se @pdonadeo abbia voglia di aggiungere il tipo "sviluppatori"

**Reply by pdonadeo** [reply]
Facciamo nella "fase due", direi. Per ora l'obbiettivo è un motore di ricerca per giochi, non per studi di produzione.
